src/ucm/mmap/install.c

```diff
@@ -221,7 +227,7 @@ ucs_status_t ucm_mmap_test_installed_events(int events)
      * we don't check the status of events which were not successfully installed
      */
     pthread_mutex_lock(&ucm_mmap_install_mutex);
-    status = ucm_mmap_test_events(events & ucm_mmap_installed_events);
+    status = ucm_mmap_test_events(events);
```

Comment:
so this means that if ucm_test_events() is called before ucp_init(), or if registration cache is not created, the function will fail?

Reply:
right, in case if no hooks are installed & no external events are used - it will fail

Reply:
this is not what we want, it will show bogus warning for shm-only case